nginx/nginx.tmpl

```diff
@@ -42,8 +42,8 @@ http {
     real_ip_recursive on;
 
     # Log format tracking timings
-    log_format upstream_info '$remote_addr - $remote_user [$time_local] '
-                             '"$request" $status $body_bytes_sent '
+    log_format upstream_info '$remote_addr - $remote_user [$time_iso8601] '
+                             '"$request" $status{{.AccessLogHeaders}} $body_bytes_sent'
```

Comment:
Shouldn't do, each iteration of prepending $http_ to each header also prepends a space